reference/api/search.md

```diff
@@ -453,11 +453,11 @@ You can then use the filter in a search query:
 
 <CodeSamples id="faceted_search_walkthrough_filter_1" />
 
-#### Filtering results with `_geoRadius`
+#### Filtering results with `_geoRadius` and `_geoBoundingBox`
 
-If your documents contain `_geo` data, you can use the `_geoRadius` built-in filter rule to filter results according to their geographic position.
+If your documents contain `_geo` data, you can use the `_geoRadius` and `_geoBoundingBox` built-in filter rules to filter results according to their geographic position.
 
-`_geoRadius` establishes a circular area based on a central point and a radius. Results beyond this area will be excluded from your search. This filter rule requires three parameters: `lat`, `lng` and `distance_in_meters`.
+`_geoRadius` establishes a circular area based on a central point and a radius. This filter rule requires three parameters: `lat`, `lng` and `distance_in_meters`.
```

Comment:
You can ignore this if you want.
How about we put `_geoRadius` and `_geoBoundingBox` in tabs? It will be easier to read and compare than scrolling
![Screen Shot 2023-03-16 at 15 36 07](https://user-images.githubusercontent.com/90181761/225605181-7e36d1f0-0a40-428d-8c90-baa62221c272.png)

Reply:
Hmmm…not a big fan of tabs, but it's a good idea. I'll do it. 

We probably need to review this page considering how much content we have to cram into it, but obviously completely out of scope for this issue.